### ISSUE TITLE
This package wildly slows down the dev experience for panda CSS. 

### ISSUE THREAD

**Issue by iyifr** [milandekruijf/pandacss-preset-radix-colors#3]
This causes a non-trivial increase in compile times in development. Any idea why ?

**Comment by milandekruijf** [comment]
Make sure you provide the `colorScales` property to the preset to only include the colors that you use. That should greatly improve compile times. Also try out using lightningcss by setting `lightningcss` to `true` in your `panda.config.ts`.

**Comment by milandekruijf** [comment]
Afaik, PandaCSS doesn't actually detect what variables you use so everything will be included. I don't know the exact numbers of variables that are created but I am sure it's in the thousands if all color scales are included.

**Comment by iyifr** [comment]
Yeah, it's a lot of variables when you include all colors which is rarely needed. You could force the consumer to specify if they want to use all colors and mention that it will degrade performance significantly if they do in the docs. The compile times greatly improve as mentioned.
